plugins/importexport/medra/classes/MedraWebservice.inc.php

```diff
@@ -57,6 +59,23 @@ function upload($xml) {
 		$arg = "<med:contentID href=\"$attachmentId\" />";
 		return $this->_doRequest('upload', $arg, $attachment);
 	}
+	
+	
+	//
+	// Public Web Service Actions deposit to Crossref
+	//
+	/**
+	 * mEDRA deposit operation.
+	 * @param $xml
+	 * @param $language
+	 */
```

Comment:
`@return` self-documentation should be added here.

Reply:
Done.